### PR TITLE
[cleanup] Normalize timestamp and output formatting across ops scripts

### LINKED ISSUE #608: [cleanup] Normalize timestamp and output formatting across ops scripts
## Why / Context
Ops scripts currently format timestamps and status lines inconsistently, which makes cross-script automation and triage parsing harder than necessary.

## Scope
**In scope**
- Introduce shared formatting helpers for timestamp and status-line rendering.
- Apply helpers to at least two existing ops scripts.
- Preserve current semantic meaning while reducing output drift.

**Out of scope**
- Rewriting script functional logic unrelated to formatting.
- Adding external logging dependencies.

Constraints:
- Keep changes small/mergeable.
- Deterministic tests only.
- Avoid touching unrelated generated artifacts unless required.

## Acceptance Criteria (required)
- [ ] At least two ops scripts use shared formatting helpers for timestamps/status text.
- [ ] Output remains human-readable and script-parseable.
- [ ] Behavioral output changes are documented in tests or golden assertions.
- [ ] No regression in existing cron/operator usage.

## Implementation Notes / Pointers
**Likely files / modules:**
- `scripts/ops/tail_decision_traces.py`
- `scripts/ops/liveness_check.py`
- `scripts/ops/runtime_fingerprint.py`

**Related tests:**
- `tests/unit/scripts/test_tail_decision_traces.py`
- `tests/unit/scripts/test_liveness_check.py`
- `tests/unit/scripts/test_runtime_fingerprint.py`

**Edge cases to handle:**
- Timezone handling in local vs UTC displays.
- Missing event timestamps should still produce stable placeholder output.
- Formatting helper failures must not crash script execution.

## Commands (local)
- `make fmt`
- `make lint`
- `uv run pytest -q tests/unit/scripts/test_tail_decision_traces.py tests/unit/scripts/test_liveness_check.py tests/unit/scripts/test_runtime_fingerprint.py`

## PR Requirements
- PR title should match the issue.
- PR body must include: `Fixes #<issue-number>`
- CI must be green; if Agent Artifacts Freshness fails, run `uv run agent-regenerate`, then commit `var/agents/...` updates.

## Codex-Ready Checklist (for the issue creator)
- [x] Clear acceptance criteria
- [x] At least one file pointer
- [x] Commands included
- [x] No ambiguous "do the right thing" language

### PR DESCRIPTION
**Pull request by Solders-Girdles**
Fixes #608

(Automated PR created by OpenClaw PR finisher)
Branch: codex/608-cleanup-normalize-timestamp-and-output-f
Commit: ee7dc9bbf8688baa5ca5ed8d383f81bad20aaa00

### CODE REVIEW

File: scripts/ops/liveness_check.py [hunk 1]
```diff
@@ -8,6 +8,8 @@
 from datetime import datetime, timezone
 from pathlib import Path
 
+from scripts.ops import formatting
```
[the remark this answers is not in the document]
Comment (Solders-Girdles) [reply scripts/ops/liveness_check.py]: Thanks — fixed by making the `formatting` import resilient to direct script execution. These ops scripts now do:

- `try: from scripts.ops import formatting`
- `except ModuleNotFoundError: import formatting`

Verified from repo root:
- `python3 scripts/ops/liveness_check.py --help`
- `python3 scripts/ops/runtime_fingerprint.py --help`
- `python3 scripts/ops/tail_decision_traces.py --help`

Pushed in 5fb02930.
